app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+const { getAlltopics } = require("./controllers/topics-controllers");
+
+app.get("/api/topics", getAlltopics);
+
+app.all("*", (request, response) => {
+  response.status(404).send({ message: "endpoint not found" });
+});
+
```

Comment:
I would have a look here for an error handler for 500 errors (i.e. anything left over!)